client/me/connected-applications/index.jsx

```diff
@@ -1,29 +1,43 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:me:connected-applications' ),
-	bindActionCreators = require( 'redux' ).bindActionCreators,
-	connect = require( 'react-redux' ).connect;
+import createReactClass from 'create-react-class';
+import debugFactory from 'debug';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+const debug = debugFactory( 'calypso:me:connected-applications' );
 
 /**
  * Internal dependencies
  */
-var ConnectedAppItem = require( 'me/connected-application-item' ),
-	EmptyContent = require( 'components/empty-content' ),
-	MeSidebarNavigation = require( 'me/sidebar-navigation' ),
-	observe = require( 'lib/mixins/data-observe' ),
-	ReauthRequired = require( 'me/reauth-required' ),
-	twoStepAuthorization = require( 'lib/two-step-authorization' ),
-	notices = require( 'notices' ),
-	SecuritySectionNav = require( 'me/security-section-nav' ),
-	Main = require( 'components/main' ),
-	successNotice = require( 'state/notices/actions' ).successNotice;
-
-const ConnectedApplications = React.createClass( {
-
+import ConnectedAppItem from 'me/connected-application-item';
+import DocumentHead from 'components/data/document-head';
+import EmptyContent from 'components/empty-content';
+import Main from 'components/main';
+import MeSidebarNavigation from 'me/sidebar-navigation';
+import notices from 'notices';
+/* eslint-disable no-restricted-imports */
+// FIXME: Remove use of this mixin
+import observe from 'lib/mixins/data-observe';
+/* eslint-enable no-restricted-imports */
+import ReauthRequired from 'me/reauth-required';
+import SecuritySectionNav from 'me/security-section-nav';
+import twoStepAuthorization from 'lib/two-step-authorization';
+import { successNotice } from 'state/notices/actions';
+
+/* eslint-disable react/prefer-es6-class */
+// FIXME: Remove use of createReactClass
+const ConnectedApplications = createReactClass( {
```

Comment:
Interesting note, viewing this page shows a console warning:

```
Warning: ConnectedApplications: isMounted is deprecated. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.
```

It seems to come from a factory in `createReactClass`.

Reply:
I dug deeper, and discovered that `isMounted` is being called by the `observe` mixin. We'll have to figure out a way to rip that out soon.

Reply:
> I dug deeper, and discovered that isMounted is being called by the observe mixin.

I just looked at the `data-observe` mixin a moment ago, while working on an unrelated task, and I believe the `isMounted` check is not really needed there. The component subscribes to events in `componentDidMount` and unsubscribes in `componentWillUnmount`. So it's guaranteed that the component is always mounted when a `update` event is being fired.